src/irmin-pack/unix/dispatcher.ml

```diff
@@ -0,0 +1,255 @@
+open Import
+module Payload = Control_file.Latest_payload
+
+module type S = sig
+  module Fm : File_manager.S
+
+  type t
+
+  val v : root:string -> Fm.t -> (t, [> Fm.Errs.t ]) result
```

Comment:
This creation function is a bit odd - none of the other files has this. Also, the only caller just transforms the error into an exception anyway, so I'm not sure it's worth it.

Reply:
Tracked by https://github.com/mirage/irmin/issues/1957